packaging/Dockerfile.centos8-nmstate-dev

```diff
@@ -4,6 +4,7 @@ RUN dnf -y install dnf-plugins-core epel-release && \
     dnf config-manager --set-enabled PowerTools && \
     dnf copr enable nmstate/ovs-el8 -y && \
     dnf copr enable networkmanager/NetworkManager-1.26 -y && \
+    dnf copr enable nmstate/varlink-el8 -y && \
```

Comment:
The `python3-varlink` is shipped in RHEL/CentOS 8.

Reply:
The version shipped in RHEL/CentOS 8 is not high enough yet.

Reply:
If we are using version less than 30.1.0 we cannot using None value's. It has been fixed in version 30.1.1.
Fixed PR : https://github.com/varlink/python/commit/baab37971f887b0f34272b6dbc8c2156c081fc5a
Origin issue in varlink : https://github.com/varlink/python/issues/10